cartoframes/context.py

```diff
@@ -1643,6 +1647,11 @@ def data(self, table_name, metadata, persist_as=None, how='the_geom'):
         return self.query(query,
                           table_name=persist_as)
 
+    def _unique(self, suggested, existing):
+        while suggested in existing:
+            suggested = '_' + suggested
```

Comment:
String concatenation is a contentious topic in python (`a + b ` vs `'%s%s' % (a, b)` vs `'{0}{1}'.format(a, b)`). In general I chose to use the last one because type coercion is automatic. But it's not a big deal, especially here. Just FYI.